contracts/user-contracts/src/main/java/org/java/mentorship/contracts/user/client/UserFeignClient.java

```diff
@@ -36,4 +37,22 @@ public interface UserFeignClient {
             value = "/verify/{userId}/{token}"
     )
     Boolean verifyUser(@PathVariable("userId") Integer userId, @PathVariable("token") String token);
+
+    @RequestMapping(
+            method = RequestMethod.POST,
+            value = "/verify/{userId}"
+    )
+    Boolean resendVerificationToken(@PathVariable("userId") Integer userId);
```

Comment:
I don't think we need a Boolean response
Everything should be dictated by the response code
200 - okay
4xx - invalid request (invalid userId, invalid token, invalid password, etc)
5xx - some error occurred